### PR TITLE
feat: Add Perl introductory resources to warm newbies about Perl #8204

### LINKED ISSUE #8204: Add Perl introductory resources to warm newbies about Perl
## Refer
https://openfoodfacts.slack.com/archives/C01USC55N67/p1678700540409499?thread_ts=1678679463.020239&cid=C01USC55N67

### PR DESCRIPTION
**Pull request by jnsereko**
### What
Ref: https://openfoodfacts.slack.com/archives/C01USC55N67/p1678700540409499?thread_ts=1678679463.020239&cid=C01USC55N67

cc @yuktea @stephanegigandet 

### Related issue(s) and discussion

- Fixes #8204

### CODE REVIEW

**Comment by stephanegigandet** [comment]
Hi @jnsereko 

Thanks for the PR!

I think it would be better to create a new document how-to-learn-perl.md , and link it from how-to-quick-start-guide.md

cc @alexgarel

**Review by jnsereko — COMMENTED** [review]
how are you finding this @yuktea @stephanegigandet?

**Comment by yuktea** [comment]
looks cool!

File: docs/dev/how-to-learn-perl.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# how to learn Perl Programming Language?
```
Comment (VaiTon) [inline docs/dev/how-to-learn-perl.md:1]:
```suggestion
# How to learn Perl programming language?
```

**Comment by stephanegigandet** [comment]
> how do you find this @yuktea @stephanegigandet? Any objections to adding a `note`?

Looks good, could you also add a link to it in how-to-quick-start-guide.md ?

**Comment by jnsereko** [comment]
i have pushed some commit @yuktea @stephanegigandet 
kindly review

File: docs/dev/how-to-quick-start-guide.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ First setup time estimate is `~10min` with the following specs:
 
 Docker provides an isolated environment, very close to a Virtual Machine. This environment contains everything required to launch the Open Food Facts server. There is **no need to install** Perl, Perl modules, Nginx, nor Apache separately.
 
+> **_NOTE:_**  New to Perl, checkout [how to learn perl](how-to-learn-perl.md).
```
Comment (VaiTon) [inline docs/dev/how-to-quick-start-guide.md:18]:
```suggestion
> **_NOTE:_**  New to Perl? Check [how to learn perl](how-to-learn-perl.md)!
```

File: docs/dev/how-to-use-repl.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # How to use Perl REPL (re.pl)
 
+> **_NOTE:_**  New to Perl, checkout [how to learn perl](how-to-learn-perl.md).
```
Comment (VaiTon) [inline docs/dev/how-to-use-repl.md:3]:
```suggestion
> **_NOTE:_**  New to Perl? Check [how to learn perl](how-to-learn-perl.md)!
```

File: docs/dev/how-to-learn-perl.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# How to learn Perl programming language?
+
+Here are some introductory resources to learn Perl:
+
+### Quick start
```
Comment (VaiTon) [inline docs/dev/how-to-learn-perl.md:5]:
```suggestion
### Quick start 

```

File: docs/dev/how-to-learn-perl.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# How to learn Perl programming language?
+
+Here are some introductory resources to learn Perl:
+
+### Quick start 
+- [Perl Youtube Tutorial](https://www.youtube.com/watch?v=c0k9ieKky7Q) - Perl Enough to be dangerous // FULL COURSE 3 HOURS.
+- [Perl - Introduction](https://www.tutorialspoint.com/perl/perl_quick_guide.htm) - Introduction to perl from tutorialspoint
+- [Impatient Perl](https://blob.perl.org/books/impatient-perl/iperl.pdf) - PDF document for people wintrested in learning perl.
+
+### Official Documentation
```
Comment (VaiTon) [inline docs/dev/how-to-learn-perl.md:10]:
```suggestion
### Official Documentation

```

File: docs/dev/how-to-learn-perl.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# How to learn Perl programming language?
```
Comment (VaiTon) [inline docs/dev/how-to-learn-perl.md:1]:
```suggestion
# How can I learn the Perl programming language?
```

**Review by alexgarel — APPROVED** [review]
Thanks @jnsereko !

**Pull request merged**